src/database/ProductExports/index.ts

```diff
@@ -0,0 +1,83 @@
+import dbConnection, { tableNames, execute } from '../connection';
+
+const prodExportTable = () => dbConnection()(tableNames.PRODUCT_EXPORTS);
+
+/**
+ * Represent a product export after retrieval from the database
+ */
+export interface ProdExportDb {
```

Comment:
i dont think this should have "db" after because productname isn't in the product export table?

Reply:
I think it's more like, what will the product exports database interface is gonna give. Not exactly what's in the database. Because this can't just return an ID, it's gotta give a product name somehow.

Reply:
pls fix

Reply:
I don't remember what there was to fix.